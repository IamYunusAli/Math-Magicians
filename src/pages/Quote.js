import React from 'react';
import styles from './pages.module.css';

const Quote = () => (
  <div className={styles.quote}>
    Without mathematics, there’s nothing you can do.
    Everything around you is mathematics. Everything around you is numbers.
    <br />
    — Shakuntala Devi, Indian writer and mental calculator
  </div>
);
export default Quote;
