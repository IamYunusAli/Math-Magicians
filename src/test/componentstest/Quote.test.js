import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

test('should match Quote snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
