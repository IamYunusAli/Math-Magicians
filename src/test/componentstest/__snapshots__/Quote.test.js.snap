// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Quote snapshot 1`] = `
<div
  className="quote"
>
  Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  <br />
  — Shakuntala Devi, Indian writer and mental calculator
</div>
`;
